birdhouse/components/weaver/config/canarie-api/canarie_api_monitoring.py.template

```diff
@@ -19,7 +19,7 @@ SERVICES['Weaver'] = {
         'releasenotes': 'https://github.com/crim-ca/weaver/blob/master/CHANGES.rst',
         'support': 'https://github.com/crim-ca/weaver/issues',
         'source': 'https://github.com/crim-ca/weaver',
-        'tryme': 'https://${BIRDHOUSE_FQDN_PUBLIC}/weaver/',
+        'tryme': '${BIRDHOUSE_PROXY_SCHEME}://${BIRDHOUSE_FQDN_PUBLIC}/weaver/',
```

Comment:
Just saw that other configs use `/${WEAVER_MANAGER_NAME}` rather than `/weaver` explicitly, so we might want to adjust while at it.